gp2gp-translator/src/integrationTest/resources/application.yml

```diff
@@ -26,18 +26,21 @@ amqp:
     username: ${PS_AMQP_USERNAME:}
     password: ${PS_AMQP_PASSWORD:}
     maxRedeliveries: ${PS_AMQP_MAX_REDELIVERIES:3}
+    closeTimeout: ${PS_CLOSE_TIMEOUT:10000}
```

Comment:
Are we intentionally settin the timeout less than the default of 60 seconds?

Reply:
yes, it is just to reduce close timeout, I think 10 sec is more than enough... also, wanted to test it using our perf tests...